authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/seeds/utils/ScenarioData.kt

```diff
@@ -117,7 +117,7 @@ class ScenarioData {
     fun organization(orgId: Int, createdByUserId: Int, applyer: (OrganizationRecord) -> Unit = {}) {
         OrganizationRecord().also {
             it.id = dummyDevOrganizationId(orgId)
-            it.name = "Organization $orgId"
+            it.name = dummyDevOrganizationName(orgId)
```

Comment:
Out of sheer curiosity, why did you move this into a function? You reuse `orgId` anyway so this is something that actually benefits from being inlined.

Okay, I see it's to reuse in tests so that's fine. It would also be fine to just use hardcoded strings in tests for verbosity but either way is okay.